geostore/Dockerfile

```diff
@@ -1,26 +1,22 @@
-FROM ubuntu:22.04 as build
-
-COPY geostore/pip.txt geostore/poetry.txt poetry.lock poetry.toml pyproject.toml /opt/
-WORKDIR /opt
-RUN apt-get update \
-    && apt-get install --assume-yes --no-install-recommends \
-    curl python-is-python3 python3-pip \
-    && rm -rf /var/lib/apt/lists/* \
-    && pip install --no-cache-dir --requirement=pip.txt \
-    && pip install --no-cache-dir --requirement=poetry.txt
+FROM python:3.9-slim as build
```

Comment:
Can we get the version from the `.python-version` file? One less thing to keep in sync.